src/app/standup/PageStandup.tsx

```diff
@@ -119,56 +147,95 @@ export const PageStandup = () => {
                   spacing={1}
                   overflow="scroll"
                   position="fixed"
-                  left={5}
+                  left="250px"
                   right={5}
                   h="full"
                 >
-                  <Box minW="10rem">
-                    <PopoverInput
-                      onSubmit={(value) => handleAddProject(value)}
-                      label="Nom"
-                      submitLabel="Ajouter un projet"
-                      placeholder="Saisir le nom du projet"
-                      placement="bottom-start"
-                    >
-                      <Button
-                        variant="@primary"
-                        size="sm"
-                        isLoading={isLoadingAddProject}
+                  <Stack direction="row" h="full">
+                    <Flex direction="column">
+                      <Stack
+                        padding="3"
+                        bg="gray.700"
+                        width="250px"
+                        color="yellow.500"
+                        h="full"
+                        position="fixed"
+                        left={1}
+                        borderRadius="md"
                       >
-                        <Icon icon={FiPlus} mr={1} /> Ajouter un projet
-                      </Button>
-                    </PopoverInput>
-                  </Box>
-                  {sortByIndex(projects)?.map((project, index) => (
-                    <Draggable
-                      key={project.id}
-                      draggableId={project.id}
-                      index={index}
-                    >
-                      {(provided) => (
-                        <Box
-                          ref={provided.innerRef}
-                          {...provided.draggableProps}
-                          {...provided.dragHandleProps}
-                          data-react-beautiful-dnd-draggable="0"
-                          data-react-beautiful-dnd-drag-handle="0"
-                          h="fit-content"
-                        >
-                          <SpeakerGroup
-                            project={project}
-                            speakers={speakers?.filter(
-                              (speaker) => speaker?.projectId === project?.id
-                            )}
-                            isLoading={isLoadingSpeakers}
-                            isError={isErrorSpeakers}
-                          />
-                          {provided.placeholder}
+                        <Box>
+                          <Box height="3em">
+                            <PopoverInput
+                              onSubmit={(value) => handleAddProject(value)}
```

Comment:
```suggestion
                              onSubmit={handleAddProject}
```

Ici tu peux passer juste la fonction

Reply:
cette parti du code a été réaliser par Hugo mais je peux corriger s'il le faut

Reply:
Comme tu le sens, les deux se font.